tailwind.config.js

```diff
@@ -76,11 +73,13 @@ export default {
         },
         focus: "#F8D177",
       },
+      maxHeight: {
+        calc: "calc(100vh - (2*32px + 52px + 40px))",
```

Comment:
1. Can't we have `"calc(100vh - 156px)"`. Cause `156px` is similar to `(2*32px + 52px + 40px)`. I just did the arithmetic.
2. `maxHeight` is generic name. Since this `calc` method is purely for Sidebar, maybe we can rename calc to something which is more contextual. For an example `sidebarcalc`.

This way in the appropriate `div` you can use it like `max-h-sidebarcalc`. See if that works